http/src/main/scala/org/http4s/blaze/http/RouteAction.scala

```diff
@@ -26,53 +28,86 @@ object RouteAction {
     *             before requesting another chunk. Termination is signaled by an __empty__ `ByteBuffer` as
     *             defined by the return value of `ByteBuffer.hasRemaining()`.
     */
-  def streaming(code: Int, status: String, headers: Headers)(body: () => Future[ByteBuffer]): HttpResponse = HttpResponse(
+  def Streaming(code: Int, status: String, headers: Headers)
```

Comment:
Yeah, I agree. Do you have a better idea in mind? I didn't get inspired and to some extent don't care: hopefully very few to no folks are using these for anything other than tinkering. 

My hope is for this to be solid and just complete enough to serve as a foundation for http4s proper so we don't have as much low level stuff laying around in there.